### ISSUE TITLE
Bitcoin Gold

### ISSUE THREAD

**Issue by MJMODZZ** [UnamSanctam/SilentCryptoMiner#1355]
As i saw, Bitcoin Gold is really profitable to mine. Is it possible to integrate this into the current SilentCryptoMiner?

**Comment by UnamSanctam** [comment]
No not in any easy way, and Bitcoin Gold isn't really that much more profitable than others, right now it even looks like UBQ is more profitable than Bitcoin Gold to mine on most cards.

**Comment by MJMODZZ** [comment]
> No not in any easy way, and Bitcoin Gold isn't really that much more profitable than others, right now it even looks like UBQ is more profitable than Bitcoin Gold to mine on most cards.

But the only currencys you can mine atm is monero and Ethereum classic

**Comment by MJMODZZ** [comment]
> No not in any easy way, and Bitcoin Gold isn't really that much more profitable than others, right now it even looks like UBQ is more profitable than Bitcoin Gold to mine on most cards.

And also another question, you can make your computername as worker Name as {COMPUTERNAME}, are there other commands like these, for example {IPADRESS}?

**Comment by MJMODZZ** [comment]
and there is also something wrong in the miner, i have no stealth targets but in my taskmngr it shows me my GPU is beeing used by 0%. but i know my GPU is mining because the fans are spinning. This is what the checker is saying
```
Miners:
--cinit-algo=etchash --pool=stratums://`0x133D7682EE41b1246C408bCDF7F72B2eCA4654E0`.{COMPUTERNAME}@eu1-etc.ethermine.org:5555  --cinit-max-gpu=100 --cinit-version="3.1.0" --cinit-id="xzkomnnrpsoruonx"
GPUs:
 NVIDIA GeForce RTX 3090 Ti
Compatible GPU found: True
```

**Comment by UnamSanctam** [comment]
>But the only currencys you can mine atm is monero and Ethereum classic

No, there are many different currencies you can mine, you can find a list of the algorithms available on the main page: https://github.com/UnamSanctam/SilentCryptoMiner

>And also another question, you can make your computername as worker Name as {COMPUTERNAME}, are there other commands like these, for example {IPADRESS}?

There are `{COMPUTERNAME}`, `{USERNAME}` and `{RANDOM}`. Things like IP address would be quite "complicated" to do since a computer does not inherently know its own IP. It would have to contact some site that would tell the miner its IP to do that.

>and there is also something wrong in the miner, i have no stealth targets but in my taskmngr it shows me my GPU is beeing used by 0%. but i know my GPU is mining because the fans are spinning. This is what the checker is saying

Yes the Task Manager does not show the kind of GPU usage that the miner uses. You'd need to use something like MSI Afterburner to check.

**Comment by UnamSanctam** [comment]
Sure, but the problem is that the miner doesn't natively know its own IP, just like your computer doesn't know its own IP either. It's much better to only mine to the pool by yourself first to lower the minimum payout before you start mining with other computers.

**Comment by MJMODZZ** [comment]
> Sure, but the problem is that the miner doesn't natively know its own IP, just like your computer doesn't know its own IP either. It's much better to only mine to the pool by yourself first to lower the minimum payout before you start mining with other computers.

**Comment by UnamSanctam** [comment]
>cmd command ipconfig Shows IPv4 adreys

Only your internal IP address, not your external one.
